### PR TITLE
add cards in site

### PR DESCRIPTION
**Pull request by DanielLima-dev**
add cards in site, review guys please...

### CODE REVIEW

**Review by ayrandev — APPROVED** [review]

File: images/img-frete.png
Comment (Diegoalcantaradev) [inline images/img-frete.png:1]: delete for img

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
```
Comment (Diegoalcantaradev) [inline .vscode/settings.json:2]: add in gitignore

**Review by Diegoalcantaradev — CHANGES_REQUESTED** [review]
review

**Review by Diegoalcantaradev — APPROVED** [review]
nice job brow

**Pull request merged**